src/lib/viewers/ActionSummaryViewer.ts

```diff
@@ -29,7 +38,9 @@ export class ConfigActionSummaryViewer extends AbstractActionSummaryViewer {
 		super(display);
 	}
 
-	public view(logFile: string, outfile?: string): void {
+	public viewPostExecutionSummary(logFile: string, outfile?: string): void {
+		// Start with separator to cleanly break from anything that's already been logged.
+		this.displayLineSeparator();
```

Comment:
This was missing the leading newline that all the other summary viewers have, so I went ahead and added it.